integration.md

```diff
@@ -81,30 +81,32 @@ We've previously stated that the forms presented by Klarna are eventually posted
 ```ruby
 # Handle POST requests to '/purchase'
 post '/purchase' do
+  @user = User.create(session, params[:userToken], 'user@email.com') unless @user
   purchase = Klarna::Purchase.new(
-    user_token:    params[:userToken],
-    reference:     "IA-#{params[:article_id]}",
-    name:          'Interesting Article',
+    user_token:    @user.token,
+    reference:     'subscription',
```

Comment:
@yuval-netanel I took the code as is from the source and didn't change anything in the Purchase constructor parameters. Could you please write down exactly what would you like to update in the doc?
If there's anything to change in the sample code as well - we can add the relevant ticket too.
Thanks

cc @eluxen